pkg/loader/gitcloner.go

```diff
@@ -131,3 +147,24 @@ func peelQuery(arg string) (string, string) {
 	}
 	return arg, ""
 }
+
+func handleHost(host string) string {
```

Comment:
Maybe rename this to `normalizeGithubHostSpec`

Reply:
changed this function to `normalizeGitHostSpec` since it doesn't only normalize github hosts. In the future, we may need to add more to this function for other hosts.